Dockerfile.openshift

```diff
@@ -14,25 +14,22 @@ RUN INSTALL_PKGS=" \
 	iproute iputils strace socat \
 	frr \
 	python3 \
+        tini \
         " && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS
 
 RUN dnf -y update && yum clean all && rm -rf /var/cache/yum/* && rm -rf /var/cache/yum
 
-# Add tini package
-ENV TINI_VERSION v0.19.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
-RUN chmod +x /sbin/tini
-
-
 # frr.sh is the entry point. This script examines environment
 # variables to direct operation and configure ovn
 ADD frr.sh /root/
 ADD daemons /etc/frr
 ADD frr.conf /etc/frr
 ADD vtysh.conf /etc/frr
 
-RUN chown frr:frr /etc/frr/daemons /etc/frr/frr.conf 
+RUN chown frr:frr /etc/frr/daemons /etc/frr/frr.conf
+RUN ln -s /usr/bin/tini /sbin/tini
+RUN chmod +x /sbin/tini
```

Comment:
do we need this?

Reply:
I am not sure when Idid it with u/s it needed it, we can try w/o it and if it didnt work we can add it

Reply:
:+1:

Reply:
give me a min checking quickly